pkg/signup/signup_service.go

```diff
@@ -213,3 +239,25 @@ func (s *ServiceImpl) GetSignup(userID string) (*Signup, error) {
 
 	return signupResponse, nil
 }
+
+func (s *ServiceImpl) GetUserSignup(userID string) (*crtapi.UserSignup, error) {
```

Comment:
could we add a comment to this? also wondering if we should re-use this func in `GetSignup`

Reply:
I've added a comment.  Using this in `GetSignup` doesn't seem to save us much, I think it would be about the same lines of code.

Reply:
yeah, just thought it would help with removing redundancy..